blueprint/core/blueprint_ScrollView.h

```diff
@@ -23,25 +23,70 @@ namespace blueprint
     class ScrollView : public View
     {
     public:
+        //==============================================================================
+        // Props following CSS Scrollbars spec with some omissions/additions.
+        // https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Scrollbars
+        static inline juce::Identifier overflowProp       = "overflow";
+        static inline juce::Identifier overflowXProp      = "overflow-x";
+        static inline juce::Identifier overflowYProp      = "overflow-y";
+        static inline juce::Identifier scrollBarWidthProp = "scrollbar-width";
+        static inline juce::Identifier scrollbarColorProp = "scrollbar-color";
+        static inline juce::Identifier scrollOnDragProp   = "scroll-on-drag";
+
         //==============================================================================
         ScrollView()
         {
             addAndMakeVisible(viewport);
-            // TODO: Set these based on props...
-            viewport.setScrollBarsShown(true, false);
+
+            //TODO: Make scrollbars hidden by default so that
+            //      explicit overflow settings required? Or show on
+            //      default and overflow: hidden can be applied?
+            viewport.setScrollBarsShown(false, false);
         }
 
         //==============================================================================
         void setProperty (const juce::Identifier& name, const juce::var& value) override
         {
             View::setProperty(name, value);
 
-            if (name == juce::Identifier("scrollbar-thumb-color"))
+            if (name == overflowYProp || name == overflowXProp || name == overflowProp)
+            {
+                const auto showVertical   = props[overflowYProp] == "scroll" || props[overflowProp] == "scroll";
+                const auto showHorizontal = props[overflowXProp] == "scroll" || props[overflowProp] == "scroll";
+
+                viewport.setScrollBarsShown(showVertical, showHorizontal);
+            }
+
+            if (name == scrollBarWidthProp)
             {
-                juce::Colour c = juce::Colour::fromString(props["scrollbar-thumb-color"].toString());
+                //TODO: Support CSS scrollbar-width "thin", "auto", "none" values
```

Comment:
Should we support these CSS values or do we keep this as just an integer value for scrollbar width ?

Reply:
Per my comment above, let's have the native API deal only in fixed integer values, but in the javascript layer we can allow `thin` and `auto` and map those to integer values at that layer

Reply:
Cool. Any thoughts on sensible values?

Reply:
Ermmm what's the default scrollbar width, like 5px? Maybe 5px and 3px or something? I think it's probably fairly innocuous